docs/self-hosted/config-options.md

```diff
@@ -229,6 +229,7 @@ for example. The format for adding LEVELS values is:
 | --------------------------- | -------------------------------------------------- | ------------------------------------------------------------------------------------------------------------ |
 | `SERVER_KEEP_ALIVE_TIMEOUT` | Timeout in milliseconds for socket to be destroyed | [server.keepAliveTimeout](https://github.com/nodejs/node/blob/master/doc/api/http.md#serverkeepalivetimeout) |
 | `SERVER_HEADERS_TIMEOUT`    | Timeout in milliseconds to parse HTTP headers      | [server.headersTimeout](https://github.com/nodejs/node/blob/master/doc/api/http.md#serverheaderstimeout)     |
+| `SERVER_SHUTDOWN_TIMEOUT`   | Timeout before the server is forcefully shut down  | 1000                                                                                                         |
```

Comment:
In the spirit of clarity and making it consistent with the other two timeout env variables above, lets tweak `Timeout` to `Timeout in milliseconds` 👍